loader/refresher_iface.go

```diff
@@ -0,0 +1,25 @@
+package loader
+
+type FileSystemOp int32
+
+// Filesystem operations that are monitored for changes
+const (
+	Create FileSystemOp = 1 << iota
+	Write
+	Remove
+	Rename
+	Chmod
+)
```

Comment:
Do you need the bit shift here? If not, I'd change the type to uint8 and just use "= iota"